conda-recipes/llvmdev_manylinux/build.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# based on https://github.com/AnacondaRecipes/llvmdev-feedstock/blob/master/recipe/build.sh
+
+set -x
+
+# Make osx work like linux.
```

Comment:
More details about what this does? It should be evident from the comment what the purpose of this line is.

Reply:
Since we are building only on linux here, can these lines not just be deleted?